examples/quickstart.ipynb

```diff
@@ -56,10 +56,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from expertsystem.ui import (\n",
-    "    InteractionTypes,\n",
-    "    StateTransitionManager,\n",
-    ")"
+    "from expertsystem.ui import InteractionTypes, StateTransitionManager"
```

Comment:
why does this change, where the brackets removed or did black do this automatically?

Reply:
Black formatting, apparently the line was too long. Not sure why it wasn't formatted before (haven't found a way to format notebooks from the terminal and integrate into the CI).

Reply:
hmm i see